therapy/cli.py

```diff
@@ -69,39 +80,76 @@ def update_normalizer_db(normalizer, prod, db_url, update_all,
             if len(non_sources) != 0:
                 raise Exception(f"Not valid source(s): {non_sources}")
 
+            CLI()._check_disease_normalizer(normalizers, endpoint_url)
             CLI()._update_normalizers(normalizers, db, update_merged)
 
-    def _help_msg(self):
+    def _check_disease_normalizer(self, normalizers, endpoint_url):
+        """Load Disease Normalizer data if Hemonc source included.
+
+        :param list normalizers: List of sources to load
+        :param str endpoint_url: Therapy endpoint URL
+        """
+        if 'hemonc' in normalizers:
+            db = DiseaseDatabase(db_url=endpoint_url)
+            n_sources = len({v.value for v in DiseaseSources})
+            if db.diseases.item_count == 0 or \
+                    db.metadata.item_count != n_sources:
+                msg = "Disease Normalizer not loaded. " \
+                      "Loading Disease Normalizer..."
+                logger.debug(msg)
+                click.echo(msg)
+                # Is there a better way to do this?
+                try:
+                    DiseaseCLI().update_normalizer_db(
+                        ['--update_all', '--update_merged',
+                         '--db_url', endpoint_url]
+                    )
+                except:  # noqa: E722
+                    pass
```

Comment:
imo, at the very least this should grab the exception (`except Exception as e`) and log what it is -- otherwise it could be easy to miss

Reply:
You're right. I was just accounting for disease normalizer to work which doesn't throw an exception. Will add another block.